swarmit/src/io/polyswarm/swarmit/apiclient/SwarmItApiClient.java

```diff
@@ -129,6 +134,12 @@ public static JSONObject getSubmissionStatus(String uuid) throws IOException, Cl
         } catch (URISyntaxException ex) {
             LOGGER.log(Level.SEVERE, "Invalid API URI.", ex);
             throw new IOException(ex);
+        } catch (NotAuthorizedException ex) {
+            LOGGER.log(Level.SEVERE, "Invalid API Key.", ex);
+            throw new IOException(ex);

```

Comment:
Should this be an IOException or a NotAuthorizedException

Reply:
It is catching the NotAuthorizedException for the difference in logging. 

It extends IOException, so I just explicitly throw IOException, which can take in a IOException subclass.